### PR TITLE
Add test to AddAnnotationWithLevel func for improving coverage

### PR DESCRIPTION
**Pull request by hase1128**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Add test set to improve test coverage in `staging/src/k8s.io/apiserver/pkg/admission/attributes.go`.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/admission/attributes_test.go [hunk 1]
```diff
@@ -63,3 +63,11 @@ func TestAddAnnotation(t *testing.T) {
 		"unexpected final annotations",
 	)
 }
+
+func TestAddAnnotationWithLevel(t *testing.T) {
+	attr := &attributesRecord{}
+
+	// test AddAnnotationWithLevel
+	err := attr.AddAnnotationWithLevel("podsecuritypolicy.admission.k8s.io/validate-policy", "privileged", auditinternal.LevelNone)
+	assert.Error(t, err)
```
Comment (yliaog) [inline staging/src/k8s.io/apiserver/pkg/admission/attributes_test.go]: could you add a description: assert.Error(t, err, "description")?

Comment (hase1128) [reply staging/src/k8s.io/apiserver/pkg/admission/attributes_test.go]: thanks the review. I add a description.

**Pull request closed without merging**